packages/trader/src/AppV2/Components/TradeParameters/LastDigitPrediction/last-digit-prediction.scss

```diff
@@ -0,0 +1,73 @@
+.last-digit-prediction {
+    display: flex;
+    flex-direction: column;
+    gap: var(--core-spacing-400);
+    padding: var(--core-spacing-400) var(--core-spacing-800);
+    border-radius: var(--core-spacing-400);
+    background: var(--core-color-opacity-black-75);
+    width: 100%;
+
+    &__title {
+        color: var(--core-color-opacity-black-600);
+    }
+    &__selector {
+        display: flex;
+        flex-direction: column;
+        gap: var(--core-spacing-400);
+        width: 100%;
+
+        &-row {
+            display: flex;
+            gap: var(--core-spacing-400);
+            width: 100%;
+        }
+        .digit {
+            height: var(--core-size-3300);
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            flex: 1;
+
+            button {
```

Comment:
Should we be using specific classes rather then tag name ? (Please ignore if irrelevant)

Reply:
@ahmadtaimoor-deriv this will work well since there is only 1 `button` tag to which this style will be applied. if you know how we could benefit from using the classname on it, please share :)